app/Components/RegisterSingleTree/TreeOverview.js

```diff
@@ -105,39 +107,38 @@ const SingleTreeOverview = ({ navigation }) => {
             setIsShowDate(false)
             setPLantationDate(selectedDate)
         }
+        const handleConfirm = (data) => onChangeDate(null, data)
+        const hideDatePicker = () => setShowDate(false)
 
-        return isShowDate && <DateTimePicker
+        return isShowDate && <DateTimePickerModal
+            isVisible={showDate}
             maximumDate={new Date()}
             testID="dateTimePicker1"
             timeZoneOffsetInMinutes={0}
             value={new Date(plantationDate)}
             mode={'date'}
             is24Hour={true}
             display="default"
-            onChange={onChangeDate}
+            onConfirm={handleConfirm}
+            onCancel={hideDatePicker}
         />
     }
     let filePath, imageSource
     if (inventory) {
         filePath = inventory.polygons[0]?.coordinates[0]?.imageUrl
         imageSource = filePath ? { uri: filePath } : false
     }
-     const renderDetails = ({ polygons }) => {
+    const renderDetails = ({ polygons }) => {
         let coords;
         if (polygons[0]) {
             coords = polygons[0].coordinates[0];
         }
-        let shouldEdit = inventory.status !== 'pending';
+        let shouldEdit = inventory.status == 'incomplete';
         let detailHeaderStyle = !imageSource ? [styles.detailHeader, styles.defaulFontColor] : [styles.detailHeader]
-        let detailContainerStyle = imageSource ? [{}] : [{}]
+        let detailContainerStyle = imageSource ? [styles.detailSubContainer] : [{}]
+
         return (
-            coords && <View style={styles.detailSubContainer}>
-                <View>
-                    <Text style={detailHeaderStyle}>Planting Date</Text>
-                    <TouchableOpacity disabled={!shouldEdit} onPress={() => setIsShowDate(true)}>
-                        <Text style={styles.detailText}>{new Date(plantationDate).toLocaleDateString()} {shouldEdit && <MIcon name={'edit'} size={20} />}</Text>
-                    </TouchableOpacity>
-                </View>
+            <View style={detailContainerStyle}>
                 <View>
                     <Text style={detailHeaderStyle}>LOCATION</Text>
                     <Text style={styles.detailText}>{`${coords.latitude.toFixed(5)}˚N,${coords.longitude.toFixed(5)}˚E`} </Text>
```

Comment:
This calculation for N, E is incorrect
If decimals are used, N , E is not necessary.

this needs to be converted to sexadecimal; -10 = 10 degree S;
or -10 = 10 degree West